tests/src/test/scala/com/nvidia/spark/rapids/ParquetScanSuite.scala

```diff
@@ -90,4 +90,54 @@ class ParquetScanSuite extends SparkQueryCompareTestSuite {
     frameFromParquetWithSchema("disorder-read-schema.parquet", StructType(Seq(
       StructField("c3_long", LongType),
       StructField("c1_int", IntegerType))))) { frame => frame }
+
+  /**
+   * Column schema of unsigned-int.parquet is:
+   * TODO: array nest type is blocked by issue: https://github.com/rapidsai/cudf/issues/9240
+   *
+   * message root {
+   *   required int32 a (UINT_8);
+   *   required int32 b (UINT_16);
+   *   required int32 c (UINT_32);
+   *   required group g {
+   *     required int32 c1 (UINT_8);
+   *     required int32 c2 (UINT_16);
+   *     required int32 c3 (UINT_32);
+   *   }
+   *   required group m1 (MAP) {
+   *     repeated group key_value {
+   *       required int32 key (UINT_8);
+   *       optional int32 value (UINT_8);
+   *     }
+   *   }
+   *   required group m2 (MAP) {
+   *     repeated group key_value {
+   *       required int32 key (UINT_16);
+   *       optional int32 value (UINT_16);
+   *     }
+   *   }
+   *   required group m3 (MAP) {
+   *     repeated group key_value {
+   *       required int32 key (UINT_32);
+   *       optional int32 value (UINT_32);
+   *     }
+   *   }
+   *   optional group m4 (MAP) {
+   *     repeated group key_value {
+   *       required int32 key (UINT_32);
+   *       required group value {
+   *         required int32 c1 (UINT_8);
+   *         required int32 c2 (UINT_16);
+   *         required int32 c3 (UINT_32);
+   *       }
+   *     }
+   *   }
+   * }
+   *
+   */
+  testSparkResultsAreEqual("Test Parquet unsigned int: uint8, uint16, uint32",
+    frameFromParquet("unsigned-int.parquet"),
+    assumeCondition = (_ => (isSpark320OrLater, "Spark version not 3.2.0+"))) {
```

Comment:
Is it necessary to preclude this test for Spark < 3.2?  Will the CPU version throw an exception when it sees an unsigned type?  Would be nice to have a comment explaining why this needs to be filtered based on Spark version if it's necessary.

Reply:
done, added comment. CPU version throw an exception when it sees an unsigned type.